lib/src/components/NxProgressBar/NxProgressBar.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { faExclamationCircle, faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+import classnames from 'classnames';
+import React, { forwardRef } from 'react';
+
+import { useUniqueId } from '../../util/idUtil';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, propTypes } from './types';
+
+import './NxProgressBar.scss';
+
+const NxProgressBar = forwardRef<HTMLProgressElement, Props>(
+    function NxProgressBar(props, ref) {
+      const {
```

Comment:
Extra native props including `className` and `id` should be handled

Reply:
> Extra native props including `className` and `id` should be handled

Fixed: 19a5769c8bc492f419e07360d86b750655c95708